content/en/getting-started/installation.md

```diff
@@ -1,155 +1,206 @@
+$ localstack --version
+{{< localstack-latest-version >}}
 {{< / command >}}
 
-You can decide to update the CLI itself, the LocalStack Docker images, or all at once:
+You are all set!
+To use all of LocalStack's features we recommend to [get a LocalStack account and set up your api key]({{< ref "api-key" >}}).
+Afterwards, check out our [Quickstart guide]({{< ref "quickstart" >}}) to start your local cloud!
 
-{{< command >}}
-$ # Update all components
-$ localstack update all
-$ # Only update the LocalStack docker images
-$ localstack update docker-images
-$ # Only update the LocalStack CLI
-$ localstack update localstack-cli
-{{< / command >}}
-
-#### Starting LocalStack with the LocalStack CLI
+## Alternatives
+Besides using the CLI, there are other ways of starting and managing your LocalStack instance:
 
-By default, LocalStack is started inside a Docker container by running:
+- [LocalStack Cockpit]({{< ref "#localstack-cockpit" >}})\
+  Get a desktop experience and work with your local LocalStack instance via the UI.
 
-{{< command >}}
-$ localstack start
-{{< / command >}}
+- [LocalStack Docker Extension]({{< ref "#localstack-docker-extension" >}})\
+  Use the LocalStack extension for Docker Desktop to work with your LocalStack instance.
 
-{{< alert title="Notes" >}}
-- This command loads all services provided by LocalStack, they will however be started on the first request reaching this service.
+- [Docker]({{< ref "#docker" >}})\
+  Use the `docker` CLI to manually start the LocalStack Docker container.
 
-- By default, LocalStack uses the image tagged `latest` that is cached on your machine, and will **not** pull the latest image automatically from Docker Hub (i.e., the image needs to be pulled manually if needed).
+- [Docker-Compose]({{< ref "#docker-compose" >}})\
+  Use `docker-compose` to configure and start your LocalStack Docker container.
 
-- The default image `localstack/localstack` in DockerHub refers to the community version of LocalStack. The `localstack/localstack-pro` image refers to the Pro version of LocalStack. Previously we maintained `localstack-light` and `localstack-full` images which has been deprecated and will be removed with the LocalStack 2.0 release.
+- [Helm]({{< ref "#helm" >}})\
+  Use `helm` to create a LocalStack deployment in a Kubernetes cluster.
 
-- When an [API key]({{< ref "api-key" >}}) is set in your environment, `localstack start` will attempt to start LocalStack with the `localstack/localtack-pro` image.
-  A successful key activation is required to start LocalStack Pro but can be disabled by setting `ACTIVATE_PRO=0`.
-{{< /alert >}}
+LocalStack runs inside a Docker container, and the above options are different ways to start and manage the LocalStack Docker container.
+For a comprehensive overview of the LocalStack images, check out our [Docker images documentation]({{< ref "docker-images" >}}).
 
 ### LocalStack Cockpit
 
 See [LocalStack Cockpit]({{< ref "cockpit" >}}).
```

Comment:
It is a bit awkward that the ToC links to this section without any context (the same applies to Docker Extension). Adding a summary sentence is directly actionable (install link) and clarifies the context:

> Download our desktop client at [localstack.cloud/products/cockpit](https://localstack.cloud/products/cockpit). See [LocalStack Cockpit]({{< ref "cockpit" >}}).

Reply:
That's a good point, I applied your suggestion. 🚀